Makefile.authx

```diff
@@ -33,8 +31,7 @@ clean-vault:
 
 #<<<
 clean-opa:
-	cat $(DIR)/lib/compose/docker-compose.yml $(DIR)/lib/logging/$(DOCKER_LOG_DRIVER)/docker-compose.yml \
-		$(DIR)/lib/opa/docker-compose.yml | docker-compose -f - down
+	docker-compose -f $(DIR)/lib/compose/docker-compose.yml docker-compose -f $(DIR)/lib/opa/docker-compose.yml down || true
```

Comment:
need to remove the second docker-compose command: should be `docker-compose -f $(DIR)/lib/compose/docker-compose.yml -f $(DIR)/lib/opa/docker-compose.yml down || true`

Reply:
Done